docs/docs/quickstart/run-workflow-on-a-schedule/README.md

```diff
@@ -0,0 +1,77 @@
+# Run workflow on a schedule
```

Comment:
Could you expand on [these docs](https://pipedream.com/docs/workflows/events/cold-starts/) and simply link to this section? I'd prefer to improve the existing docs with a better example vs. duplicating this content.

Reply:
Done. I updated the example and removed the explanation about cold starts (and replaced with a link to the existing section in the docs).